fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/svarinn/SvarInnMessage.java

```diff
@@ -1,9 +1,11 @@
 package no.difi.meldingsutveksling.ks.svarinn;
 
+import com.google.common.base.Strings;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.base.Strings'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)